src/components/socialLogin/tests/__snapshots__/FacebookButton.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Facebook Button Tests should not regress 1`] = `ShallowWrapper {}`;
```

Comment:
Why is it not rendering anything?

Reply:
@adesege this test is a bit weird. When I use test renderer it fails but with enzyme { shallow } it passes.